aspnetcore/blazor/get-started.md

```diff
@@ -54,34 +56,30 @@ In a few steps, get started with Blazor:
       ```console
       dotnet new blazor -o WebApplication1
       ```
+
+      > [!NOTE]
```

Comment:
Can we move this next to the instructions for calling `dotnet new blazorserverside`?

Reply:
Not sure I follow u: This is the only tab that they'll see for this scenario. I think we should remove the serverside instruction tho ... that makes no sense here at this time. I can add that back at Pre5.

Reply:
Also, should we reverse the order so that `dotnet new blazor` is first?

Reply:
I just removed the serverside bit ... makes no sense on this tab at this time. I updated the note to match the scenario.